test/reactTags.test.js

```diff
@@ -494,4 +495,23 @@ describe('Test ReactTags', () => {
       expect(actual).to.have.deep.members([{ id: 'Ea', text: 'Ea' }]);
     });
   });
+
+  test('Utilizes custom labelField properly', () => {
+    const testLabelField = 'name';
+    const mapper = (a) => ({ id: a.id, [testLabelField]: a.text });
+    const suggestions = defaults.suggestions.map(mapper);
+    const tags = defaults.tags.map(mapper);
+    const expectedText = tags[0][testLabelField];
+
+    const props = {
+      labelField: testLabelField,
+      suggestions,
+      tags,
+    };
+
+    const $el = mount(mockItem(props));
+    expect($el.prop('labelField')).to.equal(testLabelField);
```

Comment:
this can be removed as we are passing the same prop and checking it.
instead lets simulate `keyDown` to check if correct tag is added.
You can check this for writing the test https://github.com/josh-degraw/react-tags/blob/175887f22d908b2a7a9514ae88dc32571249506f/test/reactTags.test.js#L242